extensions/interactions/base_test.py

```diff
@@ -150,6 +148,32 @@ def test_interaction_properties(self):
             'default_value': 1,
         }])
 
+    def test_interaction_rules(self):
+        def _check_num_interaction_rules(interaction_id, expected_num):
+            interaction = interaction_registry.Registry.get_interaction_by_id(
+                interaction_id)
+            self.assertEqual(len(interaction.rules), expected_num)
+
+        _check_num_interaction_rules('MultipleChoiceInput', 1)
+        _check_num_interaction_rules('NumericInput', 7)
+        _check_num_interaction_rules('Continue', 0)
+        with self.assertRaises(KeyError):
+            _check_num_interaction_rules('FakeObjType', 0)
+
+    def test_interaction_rule_descriptions(self):
```

Comment:
Also, can we move these (and related) tests to the frontend, instead? Seems a bit awkward to validate rules in the backend when they are fully defined in the frontend.